server/routes.js

```diff
@@ -179,6 +181,15 @@ export default (app) => {
   app.post('/groups/:groupid/expenses/:expenseid/pay', auth.mustHaveRole(roles.HOST), expenses.pay); // Pay an expense.
 
   /**
+   * Comments
+   */
+  app.get('/groups/:groupid/comments', mw.paginate(), mw.sorting({key: 'createdAt', dir: 'DESC'}), comments.list); // Get latest comments for a collective
+  app.get('/groups/:groupid/expenses/:expenseid/comments', mw.paginate(), mw.sorting({key: 'createdAt', dir: 'DESC'}), comments.list); // Get latest comments for an expense
```

Comment:
Shouldn't we get all comments for an expense, instead of latest? And do it from earliest to latest? Frontend can then decide how to show it. Most expenses will likely have few comments, so that should be fine for now.